### PR TITLE
Update and fix SDK tests

### PR DESCRIPTION
**Pull request by esme**
## Description

Jira Ticket: CALL-3266

- Add unit tests for callCompleted and outgoingCall
- Update initialized payload to include `sdkVersion`
- Publish only whitelisted files to npm

## Merge Checklist

| Q                        | A 
| ------------------------ | --------------
| Adds Documentation?      |
| Any Dependency Changes?  |
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |

[BRAVE Checklist](https://github.com/HubSpot/calling-extensions-sdk/blob/master/SHIP_WITH_CARE.md)

- [x] I have read the BRAVE checklist and confirmed if the following is necessary.

| Q                              | A 
| ------------------------------ | --------------
| Backwards Compatible?          |
| Rollout/Rollback Plan?         | Publish and update dependency to `0.0.13-alpha.0`
| Automated test coverage?       | Unit tests
| Verified that changes work?    | :white_check_mark: Used the [npm pack](https://docs.npmjs.com/cli/pack) command to generate a tarball
| Expect Dependencies to Fail?   |

image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
     "test": "cross-env NODE_ENV=test npm run test:build && jasmine-browser-runner runSpecs --config=test/support/jasmine-browser.json --browser=headlessChrome"
   },
   "main": "./dist/main.js",
+  "files": [
+    "dist/"
```
Comment (esme) [inline package.json:31]: Files included in package by default are `package.json`, `README.md`, and `LICENSE`.
https://docs.npmjs.com/cli/v7/commands/npm-publish

Should we include demo/bin here?

File: src/CallingExtensions.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import IFrameManager from "./IFrameManager";
 import { messageType, errorType } from "./Constants";
-import { version } from "../package.json";
+import packageInfo from '../package.json';
```
[the remark this answers is not in the document]
Comment (esme) [reply src/CallingExtensions.js:5]: This fixes the build warning as the named export is deprecated in Webpack 5. https://webpack.js.org/migrate/5/#cleanup-the-code
```
Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)
```

File: demo/package.json [hunk 3]
```diff
@@ -5,7 +5,8 @@
   "private": true,
   "scripts": {
     "build": "node scripts/support_node_17+.js webpack",
-    "start": "node scripts/support_node_17+.js webpack-dev-server --open"
+    "serve": "node scripts/support_node_17+.js webpack-dev-server --open",
+    "start": "npm run build && npm run serve"
```
[the remark this answers is not in the document]
Comment (esme) [reply demo/package.json:9]: `webpack-dev-server --open` doesn't generate the compiled file in the `bin` folder. I'm adding this script so we can run `npm start` from the demos folder

**Review by alonso-cadenas — APPROVED** [review]

File: demo/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (esme) [reply demo/package.json]: Nvm I just tested everything again and it turns out webpack serves the content from the `bin` directory and returns a `404` route when the files are missing

**Pull request merged**